.github/workflows/ci.yml

```
@@ -0,0 +1,10 @@
+name: CI
+on: [pull_request]
```

mmm actually, sorry, but we also need master!
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```